src/main/java/no/nav/pto/veilarbportefolje/opensearch/OpensearchQueryBuilder.java

```diff
@@ -281,11 +293,29 @@ static void sorterFodeland(SearchSourceBuilder searchSourceBuilder, SortOrder or
     static void sorterStatsborgerskap(SearchSourceBuilder searchSourceBuilder, SortOrder order) {
         searchSourceBuilder.sort("hovedStatsborgerskap.statsborgerskap", order);
     }
-    
+
     static void sorterStatsborgerskapGyldigFra(SearchSourceBuilder searchSourceBuilder, SortOrder order) {
         searchSourceBuilder.sort("hovedStatsborgerskap.gyldigFra", order);
     }
 
+    static void sorterTolkebehov(Filtervalg filtervalg, SearchSourceBuilder searchSourceBuilder, SortOrder order) {
+        if (filtervalg.harTalespraaktolkFilter()) {
+            searchSourceBuilder.sort("talespraaktolk", order);
+        }
+        if (filtervalg.harTegnspraakFilter()) {
+            searchSourceBuilder.sort("tegnspraaktolk", order);
+        }
+    }
+
+    static void sorterTolkeSpraak(Filtervalg filtervalg, SearchSourceBuilder searchSourceBuilder, SortOrder order) {
```

Comment:
this is the samme method as the one above

Reply:
But does sorting care about filtervalg.harTalespraaktolkFilter()?
Should it not sort on the field that front end asks about independent of the filter that is in use?

Reply:
same with: harTegnspraakFilter
if the method is called, then maybe we should sort on the requested field?

Reply:
Hei, lets check this tomorrow morning...it will be easier to find some solution in huddle then here...